### ISSUE TITLE
[RFC] Allow setting headless as CLI param

### ISSUE THREAD

**Issue by JarJak** [TheSoftwareHouse/Kakunin#132]
Usually I run tests in headless mode, but sometimes I want to check what is going on in browser. I would be nice to have an ability to replace headless config param in CLI. Then I could do:
```
npm run kakunin -- --headless false
npm run kakunin -- --tags @wip --headless false
```

**Comment by sethii** [comment]
This is something we talked with @tgorskievo about. We're going to ship it with 3.0.0 - probably at the end of feb

**Comment by tgorskievo** [comment]
It's implemented in versions above 3.0.x
